dm/tests/safe_mode/run.sh

```diff
@@ -206,6 +206,9 @@ function safe_mode_duration() {
 	dmctl_operate_source create $WORK_DIR/source2.yaml $SOURCE_ID2
 
 	dmctl_start_task "$cur/conf/dm-task-safe-mode-duration.yaml" "--remove-meta"
+	# Worker1 need to start relay, so it is slow.
+	# It is possible that worker1 has not loaded before check diff.
+	sleep 3
```

Comment:
I don't think sleep can make it stable, you may use check_log_contains_with_retry to make sure test enter a centain state.